contrib/endpoints/src/api_manager/context/request_context.cc

```diff
@@ -20,6 +20,14 @@
 #include <uuid/uuid.h>
 #include <sstream>
 
+extern "C" {
+#include "grpc/grpc.h"
+#include "grpc/support/alloc.h"
+#include "grpc/support/log.h"
+#include "grpc/support/string_util.h"
+#include "grpc/support/sync.h"
+}
+
```

Comment:
Removed them. After changing to std::string based on Wayne's comments, I did not need them.